component/pyroscope/scrape/target.go

```diff
@@ -271,10 +272,6 @@ func populateLabels(lset labels.Labels, cfg Arguments) (res, orig labels.Labels,
 		return nil, nil, errors.New("no address")
 	}
 
-	if v := lset.Get(model.AddressLabel); v == "" {
-		return nil, nil, fmt.Errorf("no address")
-	}
-
```

Comment:
Is there a reason this is being removed? Isn't an `__address__` label still required so pyroscope.scrape knows where to collect profiles from?

Reply:
Ah, sorry, I missed that. Thanks!